index.html

```diff
@@ -70,5 +70,220 @@ <h3>LET'S CONNECT</h3>
 				</div>
 			</section>
 		</main>
+
+		<section class="works" id="portfolio">
+			<div class="works__card">
+				<img src="assets/img/Project1.svg" alt="project one" />
+				<h2>Tonic</h2>
+				<ul class="works__card--info">
+					<li><span>CANOPY</span></li>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>Back End Dev</p>
+					</li>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>2015</p>
+					</li>
+				</ul>
+				<p>
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<ul class="works__card--tags">
+					<li>HTML</li>
+					<li>CSS</li>
+					<li>JavaScript</li>
+				</ul>
+				<button type="button">See project</button>
+			</div>
+			<div class="works__card">
+				<img src="assets/img/Project2.svg" alt="project two" />
+				<h2>Multi-Post Stories</h2>
+				<ul class="works__card--info">
+					<li><span>CANOPY</span></li>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>Back End Dev</p>
+					</li>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>2015</p>
+					</li>
+				</ul>
+				<p>
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<ul class="works__card--tags">
+					<li>HTML</li>
+					<li>CSS</li>
+					<li>JavaScript</li>
+				</ul>
```

Comment:
- [ ]  Please kindly implement the styling: Active, Hover and Pressed on your see projects buttons. this will align with the figma design. nice job

Reply:
@tufoinnkuo10 thank you very much. Yes, you are right I forgot to add a couple of styles for the buttons. now they have a default style, for hover, when it is pressed and I also included the style for when it is disabled according to the Figma design.